plugins/source/aws/resources/services/ec2/network_interfaces.go

```diff
@@ -63,7 +63,7 @@ func resolveNetworkInterfaceArn(_ context.Context, meta schema.ClientMeta, resou
 		Partition: cl.Partition,
 		Service:   "ec2",
 		Region:    cl.Region,
-		AccountID: cl.AccountID,
+		AccountID: aws.ToString(item.OwnerId),
```

Comment:
Now that the `AccountID` is no longer guarenteed to be the same as `cl.AccountID` doesn't that mean for all of these resources we should change the `client.DefaultAccountIDColumn(false)` to be `true`?

Reply:
Yeah, makes sense

Reply:
acecb5c4d9cfce63945f5620c915fe9425c6dd9b